### PR TITLE
Add doc for compensation requests and voting

### PR DESCRIPTION
**Pull request by m52go**
This doc is fairly short, but let me know if it would be better to include a more brief overview at the start.

### CODE REVIEW

File: compensation.adoc [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+ - Comments that help explain what the work is, why it is valuable, etc.
+
+**For now, until the DAO launch, we assume 1 BSQ = 1 USD**.
+
+[NOTE]
+.How do I know how much to request?
+====
+Bisq contributions are valued _on completion_. The metric for that is _merged to master_ or otherwise _already delivered_ if the contribution is not code. How you determine the value you request is up to you, but it should be based on the value of the contribution you made, not on the raw time you spent.
+
+For example, it might be reasonable to request 100 BSQ for fixing some typos in a doc. But requesting 1000 BSQ for that same task, just because it took you a few hours to read through it, will probably be rejected.
```
Comment (ManfredKarrer) [inline compensation.adoc]: Maybe add something like:
If you are unsure about the estimation for the value of your contribution ask youself what you would charge for that work if you would have done it for any company as freelancer.

**Comment by m52go** [comment]
Only material change I made was implement ManfredKarrer's suggestion, so I will merge this now.

This will allow https://github.com/bisq-network/bisq-docs/pull/108 to be merged as soon as it's reviewed.

File: compensation.adoc [hunk 2]
```diff
@@ -40,7 +40,9 @@ This is what the GitHub issue is for. It should go in the https://github.com/bis
 [NOTE]
 .How do I know how much to request?
 ====
-Bisq contributions are valued _on completion_. The metric for that is _merged to master_ or otherwise _already delivered_ if the contribution is not code. How you determine the value you request is up to you, but it should be based on the value of the contribution you made, not on the raw time you spent.
+Bisq contributions are valued _on completion_. The metric for that is _merged to master_, or _already delivered_ if the contribution is not code. How you determine the value you request is up to you, but it should be based on the value of the contribution you made, not on the raw time you spent.
+
+A good rule of thumb, if you are unsure about the the value of your contribution: consider what you would charge for the work if you did it as a freelancer.
```
Comment (ManfredKarrer) [inline compensation.adoc]: "the the value" -> "the value"

Comment (m52go) [reply compensation.adoc]: Yikes, thanks. Fixed with https://github.com/bisq-network/bisq-docs/pull/110.

**Pull request merged**